package.json

```diff
@@ -32,10 +32,15 @@
   "license": "MIT",
   "homepage": "https://github.com/herbsjs/gotu#readme",
   "dependencies": {
-    "suma": "^1.0.5"
+    "suma": "^1.0.6"
   },
   "devDependencies": {
-    "mocha": "^7.1.1",
-    "nyc": "^15.0.1"
+    "mocha": "^7.2.0",
+    "nyc": "^15.1.0",
+    "eslint": "^6.8.0",
```

Comment:
I'm still not sure about how eslint is reinforced. does it depends on vs code?

Reply:
You still need a configuration file that is there in the todo-list-on-herbs repository.

I believe that we should put this file somewhere as a reference in the contribution rules file.

But if you want I can remove these files from the dev dependency.

Reply:
is there a way to automate it? 
like not letting people create a PR if the rules for lint are not been applied? 
like applying prettier automatically before a push?

Reply:
btw, there should not be a dependency to todo-list-on-herbs

Reply:
I will check to see if there is something like that and return as a new issue.

For now I will remove the PR dependencies